stdlib/public/core/Sort.swift

```diff
@@ -146,9 +146,6 @@ where Self: RandomAccessCollection, Element: Comparable {
   /// `Comparable` protocol by calling this method. Elements are sorted in
   /// ascending order.
   ///
-  /// The sorting algorithm is not stable. A nonstable sort may change the
```

Comment:
I'd suggest leaving this in for now. If this sort is better than the current one, we can just land it as an implementation improvement. But officially committing to stability needs an evolution proposal.

Reply:
Ah, good point.

Reply:
Well, it might be misleading to say it's "not stable" when it is: I'd suggest maybe wording it something like "is not guaranteed to be stable" or "may or may not be stable," or maybe explicitly, "It is an implementation detail whether the sorting algorithm is stable."

Reply:
Yeah, "not guaranteed stable" is the right phrasing.